Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/Cql2ElmVisitor.java

```diff
@@ -921,6 +921,9 @@ public NamedTypeSpecifier visitNamedTypeSpecifier(cqlParser.NamedTypeSpecifierCo
         String identifier = getTypeIdentifier(qualifiers, parseString(ctx.referentialOrTypeNameIdentifier()));
 
         DataType resultType = libraryBuilder.resolveTypeName(modelIdentifier, identifier);
+        if (null == resultType) {
+            throw new CqlCompilerException(String.format("Could not find type for library: %s and name: %s", modelIdentifier, identifier));
```

Comment:
Issue: should be `model` instead of `library`

Reply:
Fixed.